middlewares/addPrefix.go

```diff
@@ -11,7 +12,9 @@ type AddPrefix struct {
 }
 
 func (s *AddPrefix) ServeHTTP(w http.ResponseWriter, r *http.Request) {
-	r.URL.Path = s.Prefix + r.URL.Path
+	newPath := s.Prefix + r.URL.Path
+	log.Debugf("Adding Prefix %s to %s = %s", s.Prefix, r.URL.Path, newPath)
```

Comment:
Hello @emilevauge, I definitely can, I just added it because I thought it would be very useful to see in the debug mode (in production) to see how the modifiers get applied.  It actually helped a lot while testing the config to make sure it works properly, and once deployed in prod, the debug mode is typically silenced, so the logs are not too verbose.  But if you think that's not necessary, I'll surely take it out.  Just let me know what your thoughts are on this.